config-react/local_orthanc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "servers": {
+    "dicomWeb": [
+      {
+        "name": "Orthanc",
+        "wadoUriRoot": "http://localhost:8042/wado",
+        "qidoRoot": "http://localhost:8042/dicom-web",
+        "wadoRoot": "http://localhost:8042/dicom-web",
+        "qidoSupportsIncludeField": false,
+        "imageRendering": "wadors",
+        "thumbnailRendering": "wadors",
+        "requestOptions": {
+          "auth": "orthanc:orthanc",
+          "logRequests": true,
+          "logResponses": false,
+          "logTiming": true
+        }
+      }
+    ]
+  }
+}
```

Comment:
All of our config is for react now. Can we rename the `config` folder `config-old` until we're sure we can nuke what's in it, then rename this folder `config`?

If possible, can we make this work as a `.js` file? It would allow for in-line comments to explain settings/options.

Reply:
done